### DISCUSSION TITLE
How can I try out canary/beta versions of Next.js ?

### DISCUSSION THREAD

**Discussion by awareness481** [vercel/next.js#15392] [Help] [answered]
I presume it's possible by editing the package.json with the git url of the version you want to try out but I was wondering how everybody else is doing it.

**Comment by Timer** [comment, marked as answer]
We regularly publish new canary versions. You can run `yarn add next@canary` or `npm i next@canary`.

**Comment by awareness481** [comment]
Thank you @Timer !
